x-pack/plugins/dashboard_enhanced/README.asciidoc

```diff
@@ -0,0 +1,5 @@
+
+[[dashboard-enhanced-plugin]]
+== Dashboard app enhancements plugin
+
+Adds drilldown capabailities to dashboard.  Owned by the Kibana App team.
```

Comment:
Seems to be a typo here (capabailities):
```suggestion
Adds drilldown capabilities to dashboard.  Owned by the Kibana App team.
```

Reply:
Fixed, thanks for the review!